src/Calculator/GDTextSizeCalculator.php

```diff
@@ -28,6 +28,7 @@ public function __construct()
     public function calculateWidth(string $text, int $size = self::TEXT_SIZE): float
     {
         $size = $this->convertToPt($size);
+        var_dump($this->fontPath);
```

Comment:
forgotten var_dump?

Reply:
I left this for this reason: https://github.com/box-project/box/issues/519